services/cloudfront/src/test/java/software/amazon/awssdk/services/cloudfront/url/SignedUrlTest.java

```diff
@@ -13,21 +13,27 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.services.cloudfront;
+package software.amazon.awssdk.services.cloudfront.url;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 import nl.jqno.equalsverifier.EqualsVerifier;
 import org.junit.jupiter.api.Test;
-import software.amazon.awssdk.services.cloudfront.internal.url.SignedUrl;
-import software.amazon.awssdk.services.cloudfront.url.DefaultSignedUrl;
+import software.amazon.awssdk.services.cloudfront.internal.url.DefaultSignedUrl;
 
 class SignedUrlTest {
```

Comment:
Can we add a test for generateHttpRequest()?

Reply:
test added